tests/test_prepdocslib_textsplitter.py

```diff
@@ -77,3 +78,41 @@ def test_sentencetextsplitter_split_pages():
     assert split_pages[2].page_num == 2
     assert split_pages[2].text == 'e page"}'
     assert len(split_pages[2].text) <= max_object_length
+
+
+def pytest_generate_tests(metafunc):
+    """Parametrize the test_doc fixture with all the pdf files in the test-data directory."""
+    if "test_doc" in metafunc.fixturenames:
+        metafunc.parametrize("test_doc", [pdf for pdf in Path("tests", "test-data").glob("*.pdf")])
```

Comment:
It might be helpful to put "_chinese" "_japanese" and "_english" in the filenames to make it clear to non-speakers which data file tests what. I can't tell from a quick glance what languages they are.

Reply:
(Comment is here since I can't comment on filenames)